docs/program/apis/sessions.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Sessions Management with Viam's Client SDKs"
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the Sessions Management API with Viam's Client SDKs."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The Session Management API provides support for robot session management.
+
+A Session allows a **client** to express that it is actively connected or authenticated to `viam-server` on your robot and
+supports stopping actuating components when the connection is no longer active.
+
+{{< alert title="Info" color="info" >}}
+
+### Purpose of Session Management
+
+When controlling a robot or fleet with the Viam platform, you want some kind of protocol or system-wide means to understand the presence of the clients that are talking and authenticated to each robot's `viam-server` through Viam's various [resource APIs](/program/apis/).
+
+The presence of these clients at one point in time can be referred to as a "session".
+A session technically is any presence mechanism at the RDK application layer maintained by any client with `viam-server`.
+Session management, as explained here, allows for safer robotic operation when you are dealing with moving parts on the robot (the "actuating controls").
+
+Specifically, without session management, controls that would be "sticky" (for example [`SetPower()` of a base](/components/base/#setpower)) based on the last input of a **client**, meaning the API request from the client sets the flow of electricity on a part of the robot and then does not time out, could lead to **having a robot trying to continue doing what it was told to do forever.**
+
+You do not want that to happen.
```

Comment:
Suggestion:
```suggestion
For example, imagine a wheeled rover gets a [`SetPower()`](/components/base/#setpower)) command as the last input from a client before the connection to the robot is interrupted.
Without session management, the API request from the client sets the flow of electricity to the motors of the robot and then does not time out, causing the robot to continue driving forever, colliding with objects and people.

With session management, if the robot does not receive a signal from the client at regular intervals, it will safely stop until the connection is reestablished.
```

Reply:
@sguequierre thoughts on this rework?

Reply:
This way of phrasing it is really good/a lot easier to understand, sorry @ delay -- I was leaving this up to read through properly during last pass 😆 will implement now